docs/CONTRIBUTING.md

```diff
@@ -109,21 +109,23 @@ sudo add-apt-repository \
 6. Update the apt package index again: `sudo apt-get update`
 7. Install the latest version of Docker Engine community: `sudo apt-get install docker-ce docker-ce-cli containerd.io`
 8. Add yourself to the Docker group ([source](https://docs.docker.com/install/linux/linux-postinstall/)):
-    1. Create the Docker group: `groupadd docker`
-    1. Add your user to the group: `sudo usermod -aG docker $USER`
-    1. Log out and log back in (you may have to restart if you are running through a virtual machine)
+   1. Create the Docker group: `groupadd docker`
+   1. Add your user to the group: `sudo usermod -aG docker $USER`
+   1. Log out and log back in (you may have to restart if you are running through a virtual machine)
 9. Verify your installation by running `docker run hello-world`. This should print a hello world paragraph that includes a confirmation that Docker is working on your system.
-_NOTE: This may cause errors if you have already tried to run docker before. If you get errors then run the following commands to reset it:_
+   _NOTE: This may cause errors if you have already tried to run docker before. If you get errors then run the following commands to reset it:_
+
 ```
 sudo chown "$USER":"$USER" /home/"$USER"/.docker -R
 sudo chmod g+rwx "$HOME/.docker" -R
 ```
+
 10. Now run docker as a service on your machine, on startup:
+
     1. Enable docker on startup: `sudo systemctl enable docker`
     1. Disable docker on startup: `sudo systemctl disable docker`
-**Install Docker-Compose**
```

Comment:
I can't see this line anymore, where did it go?

Reply:
Thanks, I thought I missed something.  Added back in a new commit.